cmd/list.go

```diff
@@ -0,0 +1,280 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"sync"
+	"text/tabwriter"
+
+	"github.com/urfave/cli"
+
+	"github.com/manifoldco/torus-cli/api"
+	"github.com/manifoldco/torus-cli/apitypes"
+	"github.com/manifoldco/torus-cli/config"
+	"github.com/manifoldco/torus-cli/envelope"
+	"github.com/manifoldco/torus-cli/errs"
+	"github.com/manifoldco/torus-cli/pathexp"
+)
+
+func init() {
+	list := cli.Command{
+		Name:      "list",
+		ArgsUsage: "",
+		Usage:     "List your org's project structure and its secrets.",
```

Comment:
Perhaps something along the lines of `List allows you to list and filter all secrets you can access inside a project` ?